contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -939,8 +940,12 @@ def copy(
         channel_id = target.channel_id
 
         if ContentNode.filter_by_pk(pk=pk).exists():
-            error = ValidationError("Copy pk already exists")
-            return str(error)
+            if is_retry:
+                copy_node = ContentNode.objects.get(pk=pk)
```

Comment:
This needs to use `self.get_queryset()` otherwise I don't think this is preventing deletions of nodes that shouldn't be editable by a user